package.json

```diff
@@ -86,7 +87,7 @@
     "dotenv-cli": "^3.1.0",
     "eslint": "^7.1.0",
     "eslint-plugin-storybook": "^0.5.12",
-    "framer-motion": "^7.0.0",
+    "framer-motion": "=7.0.0",
```

Comment:
likewise

Reply:
One of the external libraries (I think framer-motion) tries and figure out whether we're running React 17 or 18 and call hooks like `useId` if they think we're on React 18.

After merging Joao's last PR that updates a lot of the footer/social links and updates the DS package, the IllustratedFeature component started to crash because the library started calling `useId`. We didn't catch this in CI because the CI doesn't run smoke tests.

I could not find the exact cause that triggers this, but freezing those two deps to the exact versions we installed helped our component stop crashing.

I don't want to investigate further and properly fix this because Kyle told me we are supposed to abandon this repo in favour of the `web` monorepo.